package.json

```diff
@@ -102,6 +102,7 @@
     "grunt-bytesize": "^0.2.0",
     "grunt-contrib-clean": "^2.0.0",
     "grunt-contrib-concat": "^1.0.1",
+    "grunt-contrib-connect": "^3.0.0",
```

Comment:
Not too keen on this solution, as we now have both http-server and grunt-connect. Should probably get rid of one of 'm. Maybe I should use `grunt-run` instead and have that start http-server?